From 33dc0b3fc6de5d7a400d24a69098ec1b23917e44 Mon Sep 17 00:00:00 2001
From: Mark Reynolds <mreynolds@redhat.com>
Date: Mon, 22 May 2017 12:25:42 -0400
Subject: [PATCH] Ticket 49157 - fix error in ds-logpipe.py

Description:  Fix typo in ds-logpipe.py

https://pagure.io/389-ds-base/issue/49157

Reviewed by: mreynolds(one line commit rule)

(cherry picked from commit 15f5f6ac42768ae0cd2040cc4169abde8187bcdf)
---
 ldap/admin/src/scripts/ds-logpipe.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ldap/admin/src/scripts/ds-logpipe.py b/ldap/admin/src/scripts/ds-logpipe.py
index 13712ea..f29a9ff 100644
--- a/ldap/admin/src/scripts/ds-logpipe.py
+++ b/ldap/admin/src/scripts/ds-logpipe.py
@@ -318,7 +318,7 @@ except OSError as e:
             sys.exit(1)
     else:
         print("Failed to create log pipe - %s [error %d]" % (e.strerror, e.errno))
-        sys.ext(1)
+        sys.exit(1)
 
 if debug:
     print("Listening to log pipe", logfname, "number of lines", maxlines)
-- 
2.9.4

